server/src/main/java/org/gluu/oxtrust/api/client/RegistrationManagementRequest.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * oxTrust is available under the MIT License (2008). See http://opensource.org/licenses/MIT for full text.
+ *
+ * Copyright (c) 2014, Gluu
+ */
+package org.gluu.oxtrust.api.client;
+
+import org.codehaus.jackson.map.DeserializationConfig;
+import org.codehaus.jackson.map.ObjectMapper;
+import org.codehaus.jettison.json.JSONException;
+import org.codehaus.jettison.json.JSONObject;
+import org.xdi.model.GluuAttribute;
+import org.xdi.oxauth.client.BaseRequest;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.codehaus.jackson.map.DeserializationConfig.Feature.FAIL_ON_UNKNOWN_PROPERTIES;
+import static org.gluu.oxtrust.model.registration.ManagementRegReqParam.CAPTCHA_DISABLED;
+import static org.gluu.oxtrust.model.registration.ManagementRegReqParam.SELECTED_ATTRIBUTES;
+import static org.xdi.oxauth.model.util.StringUtils.toJSONArray;
+
+/**
+ * Wrapper for registration management service's request.
+ *
+ * @author Shoeb Khan
+ * @version July 06, 2018
+ */
+public class RegistrationManagementRequest  {
+
+
+    private List<GluuAttribute> selectedAttributes;
```

Comment:
I think we're not supposed to use the models in the api as they are, both can evolve separately.
And please don't forget about validation if required.

Reply:
As discussed, please create a DTO for the GluuAttribute with only it's fields.